Documentation/Testing/ExtensionTesting.rst

```diff
@@ -315,12 +309,28 @@ our tests in:
 
 The package `typo3/testing-framework` that we added as `require-dev` dependency has some basic Core
 extensions set as dependency, we end up with the Core extensions `backend`, `core`, `extbase`,
-`fluid` and `frontend` in `.Build/Web/typo3/sysext`. Additionally, the
-:php:`ExtensionTestEnvironment` hook linked our git root checkout as extension into `.Build/Web/typo3conf/ext`.
-
+`fluid` and `frontend` in `.Build/Web/typo3/sysext`.
```

Comment:
Are you sure it still works like that in TYPO3 v12? I believe the packages are in vendor now

Reply:
The page needs further changes. I only wanted to remove the ExtensionTestEnvironment for now.

I think it makes more sense to make all the changes for each branch separately (or rather v11 and main+v12). That is probably better since there are bound to be several changes.

Reply:
You can also make the changes for version 11 first, backport them and then make a follow up only for version 12